dags/multipod/configs/gke_config.py

```diff
@@ -108,7 +108,7 @@ def get_gke_maxtext_nightly_config(
   current_date = current_time.strftime("%Y-%m-%d")
   current_datetime = current_time.strftime("%Y-%m-%d-%H-%M-%S")
   base_output_directory = (
-      f"{gcs_bucket.XLML_OUTPUT_DIR}/maxtext/nightly/automated/{current_date}"
+      f"{gcs_bucket.OUTPUT_DIR}/maxtext/nightly/automated/{current_date}"
```

Comment:
nit: can we clarify the difference between `OUTPUT_DIR` and `GCS_OUTPUT`? I think these are similar enough to cause confusion in the future. What do you think of calling `OUTPUT_DIR` something like `GCS_OUTPUT_BASE_DIR`?

Reply:
SG! I will update that.

Reply:
I will update it to `BASE_OUTPUT_DIR` as it comes from `gcs_bucket` file, which indicates it belongs to gcs already.